CHANGELOG.md

```diff
@@ -83,6 +83,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   selects the branch named "main", but not "maint". `description(literal:"")`
   selects commits whose description is empty.
 
+* `jj log` timestamp format now accepts `.utc()` to align all timestamps into
+  the UTC timezone.
```

Comment:
"align all timestamps" made me think that it would change all timestamps in the output, which isn't true if you do e.g. `author.timestamp().utc() ++ " " committer.timestamp()` (the committer timestamp won't be UTC). I might instead say "to convert a timestamp to UTC".